sdks/java/extensions/sql/datacatalog/src/main/java/org/apache/beam/sdk/extensions/sql/meta/provider/datacatalog/SchemaUtils.java

```diff
@@ -39,6 +39,7 @@
           .put("DATETIME", FieldType.DATETIME)
           .put("DOUBLE", FieldType.DOUBLE)
           .put("FLOAT", FieldType.DOUBLE)
```

Comment:
Actually why FLOAT maps to DOUBLE?

Reply:
Is there a 32 bits floating point type from datacatalog?

Reply:
BQ standard SQL only supports [`FLOAT64`](https://cloud.google.com/bigquery/docs/reference/standard-sql/data-types). BQ legacy SQL supports `FLOAT` that is actually double: https://cloud.google.com/bigquery/data-types . `DOUBLE` came from Data Catalog. The plan is to use `FLOAT64` as the only one supported (in BQ => DC path) and remove the other floats later. I don't know if we need to add `FLOAT32`.